src/middleware/db-connection/index.js

```diff
@@ -0,0 +1,32 @@
+//this is for testing only. Should be replaced once the real code is ready
```

Comment:
@rachael-oregan  this is only for get integration tests work. Feel free to remove this one and replace it with the middleware you are working on.